docs/stressng.md

```diff
@@ -65,3 +65,55 @@ Once done creating/editing the resource file, you can run it by:
 # kubectl apply -f <path_to_cr_file>
 ```
 
+## Running stressng in VMs through kubevirt/cnv [Preview]
+Note: this is currently in preview mode.
+you can modify [cr.yaml](../resources/crds/ripsaw_v1alpha1_stressng_vm.yaml) to your needs.
+
+
+### Pre-requisites
```

Comment:
Optional*

Reply:
Yes, test both ripsaw_v1alpha1_stressng_cr.yaml and ripsaw_v1alpha1_stressng_vm.yaml